omod/src/main/java/org/openmrs/web/servlet/QuickReportServlet.java

```diff
@@ -359,4 +362,11 @@ private String getTemplate(String reportType) {
 		
 		return template;
 	}
+	
+	private String sanitizeInput(String input) {
+		if (input == null) {
+			return null;
+		}
+		return input.replaceAll("[<>\"'%;()&+]", "");
```

Comment:
I do not like the manual sanitisation. Can't we use something like WebUtil.escapeHTML() or https://github.com/openmrs/openmrs-module-legacyui/blob/5e3108ae63e470c738d6116a186f7fc045924c51/omod/src/main/webapp/resources/scripts/openmrs.js#L690

Reply:
@dkayiwa resolved comment